src/bundle/Resources/public/js/scripts/admin.context.menu.js

```diff
@@ -21,40 +25,109 @@
         },
         onAdapted: (visibleItems, hiddenItems) => {
             const hiddenButtonsIds = [...hiddenItems].map((item) => item.querySelector('.ibexa-btn').id);
+            const topBranchItems = multilevelPopupMenu.getBranchItems(topBranch);
```

Comment:
Is branch item like all of the items in current menu?
what is topBranchItems then? ParentItems of currentBranch?
Maybe parentBranchItems?
Or only like pinned items on top?

Reply:
I think I don't understand these questions, can you be more specific/precise? :)

Reply:
I just dont understand what is topBranch items :D